app/mailers/devise/mailer.rb

```diff
@@ -1,7 +1,18 @@
 class Devise::Mailer < Devise.parent_mailer.constantize
   include Devise::Mailers::Helpers
 
-  # lock時にメールを送らない
-  def unlock_instructions(_record, _token, _opts = {})
+  def reset_password_instructions(record, token, opts = {})
```

Comment:
__[rails_best_practices]__
* use multipart/alternative as content_type of email [:ref](http://rails-bestpractices.com/posts/2010/08/05/use-multipart-alternative-as-content_type-of-email/)